### ISSUE TITLE
Problems with R-devel

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/effectsize#601]
Relevant change in R-devel: https://github.com/r-devel/r-svn/commit/c88de76f9a92702c6086deb89bb7ca2bb4a93294

cf. https://cran.r-project.org/web/checks/check_results_effectsize.html

```r
 Quitting from lines 111-116 [unnamed-chunk-8] (standardized_differences.Rmd)
    Error: processing vignette 'standardized_differences.Rmd' failed with diagnostics:
    cannot use 'paired' in formula method
```

**Comment by mattansb** [comment]
Thanks @IndrajeetPatil - it seems like from version 4.4.0 the formula interface does not allow for the `paired = TRUE` argument (I could not find any documentation of this change).

Does this need to be fixed now? How do dev errors work?

**Comment by strengejacke** [comment]
If this is not a bug, but an intended feature, you'll get an email, asking to fix CRAN errors.

**Comment by IndrajeetPatil** [comment]
I have already received an email about this for one of my packages so you should receive one soon as well.

**Comment by strengejacke** [comment]
Submission of _performance_ under check because to "changes to worse in rev-dep", however, these are false positives (for _performance_). The issue for _effectsize_, _rempsyc_ and _statsExpression_ seem to be identical.

https://cran.r-project.org/web/checks/check_results_effectsize.html
https://cran.r-project.org/web/checks/check_results_rempsyc.html
https://cran.r-project.org/web/checks/check_results_statsExpressions.html

**Comment by strengejacke** [comment]
btw, where is the page again where you could see when a function was introduced in R?

**Comment by mattansb** [comment]
https://hughjonesd.shinyapps.io/rcheology/

**Comment by IndrajeetPatil** [comment]
> Submission of _performance_ under check because to "changes to worse in rev-dep", however, these are false positives (for _performance_). The issue for _effectsize_, _rempsyc_ and _statsExpression_ seem to be identical.
> 
> https://cran.r-project.org/web/checks/check_results_effectsize.html https://cran.r-project.org/web/checks/check_results_rempsyc.html https://cran.r-project.org/web/checks/check_results_statsExpressions.html

I am submitting _statsExpressions_ update today, which fixes the issues on R-devel.

**Comment by mattansb** [comment]
I will also submit today/tomorrow.

**Comment by strengejacke** [comment]
No hurry from my side, _performance_ is on CRAN. I responded that the rev dep failures were false positives.

**Comment by rempsyc** [comment]
I really wonder why they've done this change. Indeed I cannot find any information online about this change and it seems like it is quite a breaking change. I guess it could still change since it is the development version?

**Comment by IndrajeetPatil** [comment]
> I guess it could still change since it is the development version?

No, it won't. Otherwise, they wouldn't have already sent out emails to the maintainers to make this change.

**Comment by rempsyc** [comment]
I see. Found it: It apparently fixes a bug: https://bugs.r-project.org/show_bug.cgi?id=14359

Discussed in this thread: r-devel/r-svn#142
